Pascal/Pascal.sublime-syntax

```diff
@@ -47,7 +47,7 @@ contexts:
             0: punctuation.definition.comment.pascal
           push:
```

Comment:
@keith-hall A bunch of the Textmate languages would match from the beginning of the line with whitespace before a comment line start. I feel like this has to do with how it's regex engine matched tokens, although I am far from an expert in that.

As we've been rewriting the syntaxes, I've been removing things such as this whenever possible.